applications/external/wifi_marauder_companion/scenes/wifi_marauder_scene_start.c

```diff
@@ -96,6 +96,13 @@ const WifiMarauderItem items[NUM_MENU_ITEMS] = {
       "sniffskim"},
      NO_ARGS,
      FOCUS_CONSOLE_END,
+     SHOW_STOPSCAN_TIP},
+	{"Signal Monitor",
```

Comment:
Fix indent? You can also run the formatter, but that might result in other changes: ./fbt format

Reply:
Applied fix